tests/Utilities.Mongo.UnitTests/ContextManagerTests.cs

```diff
@@ -0,0 +1,86 @@
+using FluentAssertions;
+using MatrTech.Utilities.Mongo.Models;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MongoDB.Bson;
+using MongoDB.Driver;
+using System;
+
+namespace MatrTech.Utilities.Mongo.UnitTests
+{
+    [TestClass]
+    public class ContextManagerTests
+    {
+        private const string connectionUrl = "mongodb://localhost:27017";
+
+        [TestMethod]
+        public void Create_DerivedFromMongoContext_ShouldNotBeNull()
+        {
+            var databaseName = $"{Guid.NewGuid()}";
+            ContextManager.Create<TestContext>(connectionUrl, databaseName)
+                .Should().NotBeNull();
```

Comment:
check (line)formatting in file

Reply:
What do you mean by this?

Reply:
...).Should().NotBeNull();

vs.

...)
.Should()
.NotBeNull();


consistancy in file